spec/rails_helper.rb

```diff
@@ -27,13 +28,8 @@
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include ActiveSupport::Testing::Assertions
```

Comment:
I added it when `assert_difference` was undefined. It might be the case that this module is included by default in the mini-test environment but not in Rspec.